src/systemctl/systemctl-show.c

```diff
@@ -1047,7 +1047,7 @@ static int print_property(const char *name, const char *expected_value, sd_bus_m
 
                         return 1;
 
-                } else if (STR_IN_SET(name, "SystemCallFilter", "SystemCallLog", "RestrictAddressFamilies")) {
+                } else if (STR_IN_SET(name, "SystemCallFilter", "SystemCallLog", "RestrictAddressFamilies", "RestrictFileSystems")) {
```

Comment:
this commit "core: show RestrictFileSystems= property" should be folded in the commit that adds the directive